config-overrides.js

```diff
@@ -32,6 +32,12 @@ module.exports = function override(/** @type{import("webpack").Configuration} */
     config.module.exprContextCritical = false
     config.module.unknownContextCritical = false
 
+    config.plugins.forEach(p => {
```

Comment:
Can you leave a comment to describe what does this code do?

Reply:
I think it quite obvious...